packages/transformers/webextension/src/WebExtensionTransformer.js

```diff
@@ -193,6 +193,7 @@ async function collectDependencies(
           await glob(path.join(assetDir, files[j]), fs, {})
```

Comment:
Here you can try extracting the pipeline and query strings into variables, then add them back onto `path.relative` result (rather than using the options object for `addURLDependency`, so you don't have to parse).